coral-visualization/src/test/java/com/linkedin/coral/vis/VisualizationUtilTest.java

```diff
@@ -18,27 +26,103 @@
 import com.linkedin.coral.common.HiveMscAdapter;
 import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
 
+import guru.nidi.graphviz.attribute.Font;
+import guru.nidi.graphviz.engine.Format;
+import guru.nidi.graphviz.engine.Graphviz;
+import guru.nidi.graphviz.model.Factory;
+import guru.nidi.graphviz.model.Graph;
+import guru.nidi.graphviz.model.Node;
+
 import static org.testng.Assert.*;
 
 
 public class VisualizationUtilTest {
   private static final String CORAL_VISUALIZATION_TEST_DIR = "coral.visualization.test.dir";
 
   @Test
-  public void testBasicQuery() {
+  public void testRenderToFile() {
+    String[] queries =
```

Comment:
Can you extract out the SQLs to test in a data provider like [here](https://github.com/linkedin/coral/blob/711268b04cff8057d7327f1c72c8d516461c006d/coral-trino/src/test/java/com/linkedin/coral/trino/rel2trino/HiveToTrinoConverterTest.java#L51) ?

Reply:
I do not expect this list to grow much. This is just to test the functionality of writing the image to a file. Individual tests will have their own test cases.